docs/examples/promise/06_make_transfer/index.js

```diff
@@ -1,7 +1,7 @@
 // @ts-check
 // Import the API, Keyring and some utility functions
-const { ApiPromise } = require('@polkadot/api');
-const { Keyring } = require('@polkadot/keyring');
+const { ApiPromise } = require('@cennznet/api');
+const { Keyring } = require('@cennznet/keyring');
```

Comment:
we don't have `@cennznet/keyring`, use `@cennznet/util`

Reply:
updated